tools/min-base-version

```diff
@@ -29,7 +29,7 @@ proj_dir = os.path.dirname(os.path.dirname(os.path.realpath(sys.argv[0])))
 
 max_version = 0
 
-for manifest in glob(os.path.join(proj_dir, 'pkg/*/manifest.json.in')):
+for manifest in glob(os.path.join(proj_dir, 'dist/*/manifest.json')):
```

Comment:
This likely causes the error. Either the tree isn't built at the time of `dist-hook` (when it calls tools/gen-spec-dependencies) or the build happens in a separate build directory so that it needs to happen in the current dir, not in `proj_dir`?